cmake/legacy.cmake

```diff
@@ -0,0 +1,173 @@
+          src/eventhandler/EventHandler_General.cpp
+          src/eventhandler/EventHandler_Config.cpp
+          src/eventhandler/EventHandler_Scenes.cpp
+          src/eventhandler/EventHandler_Inputs.cpp
+          src/eventhandler/EventHandler_Transitions.cpp
+          src/eventhandler/EventHandler_Filters.cpp
+          src/eventhandler/EventHandler_Outputs.cpp
+          src/eventhandler/EventHandler_SceneItems.cpp
+          src/eventhandler/EventHandler_MediaInputs.cpp
+          src/eventhandler/EventHandler_Ui.cpp
+          src/eventhandler/EventHandler.h
+          src/eventhandler/types/EventSubscription.h
+          src/requesthandler/RequestHandler.cpp
+          src/requesthandler/RequestHandler_General.cpp
+          src/requesthandler/RequestHandler_Config.cpp
+          src/requesthandler/RequestHandler_Sources.cpp
+          src/requesthandler/RequestHandler_Scenes.cpp
+          src/requesthandler/RequestHandler_Inputs.cpp
+          src/requesthandler/RequestHandler_Transitions.cpp
+          src/requesthandler/RequestHandler_Filters.cpp
+          src/requesthandler/RequestHandler_SceneItems.cpp
+          src/requesthandler/RequestHandler_Outputs.cpp
+          src/requesthandler/RequestHandler_Stream.cpp
+          src/requesthandler/RequestHandler_Record.cpp
+          src/requesthandler/RequestHandler_MediaInputs.cpp
+          src/requesthandler/RequestHandler_Ui.cpp
+          src/requesthandler/RequestHandler.h
+          src/requesthandler/RequestBatchHandler.cpp
+          src/requesthandler/RequestBatchHandler.h
+          src/requesthandler/rpc/Request.cpp
+          src/requesthandler/rpc/Request.h
+          src/requesthandler/rpc/RequestBatchRequest.cpp
+          src/requesthandler/rpc/RequestBatchRequest.h
+          src/requesthandler/rpc/RequestResult.cpp
+          src/requesthandler/rpc/RequestResult.h
+          src/requesthandler/types/RequestStatus.h
+          src/requesthandler/types/RequestBatchExecutionType.h
+          src/utils/Crypto.cpp
+          src/utils/Crypto.h
+          src/utils/Json.cpp
+          src/utils/Json.h
+          src/utils/Obs.cpp
+          src/utils/Obs_StringHelper.cpp
+          src/utils/Obs_NumberHelper.cpp
+          src/utils/Obs_ArrayHelper.cpp
+          src/utils/Obs_ObjectHelper.cpp
+          src/utils/Obs_SearchHelper.cpp
+          src/utils/Obs_ActionHelper.cpp
+          src/utils/Obs.h
+          src/utils/Obs_VolumeMeter.cpp
+          src/utils/Obs_VolumeMeter.h
+          src/utils/Obs_VolumeMeter_Helpers.h
+          src/utils/Platform.cpp
+          src/utils/Platform.h
+          src/utils/Compat.cpp
+          src/utils/Compat.h
+          src/utils/Utils.h
+          deps/qr/cpp/QrCode.cpp
+          deps/qr/cpp/QrCode.hpp)
+
+target_include_directories(
+  obs-websocket PRIVATE ${Qt5Core_INCLUDES} ${Qt5Widgets_INCLUDES} ${Qt5Svg_INCLUDES}
+                        ${Qt5Network_INCLUDES} "deps/asio/asio/include" "deps/websocketpp")
```

Comment:
Qt5?

Reply:
I made no changes to the legacy CMake file (because it's supposed to reflect the _current_ state of the files).